en/docs/learn/message-routing.md

```diff
@@ -63,19 +63,17 @@ Follow the steps below to implement the message routing service.
 
 
 ### Step 2: Create an HTTP service.
-1. In the design view, click on the **`Add Construct`** button.
-2. Select **`Service`** from the menu.
-3. Select **`HTTP Service`** from the Service Type dropdown.
-4. Select **`Design From Scratch`** option.
-5. Enter the service name as `HealthCare`, path as `/healthcare`, and `8290` as the port.
-6. Click on the **`Create Service`** button to create the new service with the specified configurations.
+1. In the design view, click on the **`Add Artifact`** button.
+2. Select **`HTTP Service`** under the **`Integration as API`** category.
+3. Select the **`+ Listeners`** option from the **`Listeners`** dropdown to add a new listener.
+4. Enter the listener name as `healthListner`, `8290` as the port and click on the **`Save`** button. 
+5. Add the service base path as `/healthcare` and select the **`Design from Scratch`** option as the **`The contract of the service`**.
```

Comment:
```suggestion
5. Add the service base path as `/healthcare` and select the **`Design from Scratch`** option as the contract of the service.
```

Reply:
`The Contract Of The Service` is the radio button's label. There it needs to be bold.

Reply:
Yeah the issue is the word is repeated in your original changes, added this suggestion to omit that